web3/contracts/StemQueue.sol

```diff
@@ -60,16 +64,16 @@ contract StemQueue is SemaphoreCore, SemaphoreGroups {
 		// }
 
 		/// @dev Allow anyone to create a new group
-		/// @param groupId: Id of the group.
 		/// @param depth: Depth of the tree.
     /// @param zeroValue: Zero value of the tree.
 		function createProjectGroup(
-				uint256 groupId,
         uint8 depth,
         uint256 zeroValue,
         address admin
     ) external {
-			  // Use current group ID and then increment
+			  // Increment groupId then use
+        _currentGroupId.increment();
+        uint256 groupId = _currentGroupId.current();
```

Comment:
This is a much better solution to use the counter for the index and just increment it.  👍